### PR TITLE
new icon: angular (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #991: [FEATURE REQUEST] Update AngularJS icon to Angular
### Problem

[Upgrading from AngularJS to Angular](https://angular.io/guide/upgrade) with new icon

### Suggested feature

[Full color logo](https://angular.io/presskit#full-color-logo)

### Why should we have this feature?

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by gabrielfins**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #991**

## Link to prove your SVG is correct and up-to-date.

https://angular.io/presskit

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey! Thanks for submitting a PR.
Could you please revert the formatting changes and make another PR for that?

Let us know if you need any help. Thanks. 😄


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: devicon.json [hunk 1]
```diff
@@ -181,6 +181,30 @@
...
+                "plain"
+            ]
+        },
+        "color": "#FFFFFF",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
+        ]
```
Comment (kilian-paquier) [inline devicon.json:206]: Hello @gabrielfins, if you include a specific plain svg, then you don't need to declare an alias between the original and the plain font.

**Comment by gabrielfins** [comment]
Is it correct now?

**Comment by kilian-paquier** [comment]
Seems good on the `devicon.json ` side.

But I think you need the `angular-plain.svg` to be a svg with only one `<path>`.

Can you confirm @Panquesito7 ?

**Comment by Snailedlt** [comment]
@kilian-paquier never heard of that requirement before :O

**Comment by kilian-paquier** [comment]
@Snailedlt My bad then ! The `paths are united` in the https://github.com/devicons/devicon/wiki/SVG-Standards seemed like the same for me 😁

**Comment by Snailedlt** [comment]
@kilian-paquier Ahh, yes you're right! Hadn't seen that there before :)

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Some small changes needed, also can we add the wordmark logo too?
https://www.vectorlogo.zone/util/preview.html?image=/logos/angular/angular-ar21.svg
angular-wordmark

This is the wordmark logo they use on their website
image
Would be nice to have exactly that one, but it's a .png and not a .svg file :/

**Comment by gabrielfins** [comment]
Hey y'all, I think all the changes requested are done now. I've added the wordmark logo that @Snailedlt requested and changed the colors to match the one they use in their website. I've also changed the plain version to have only one `<path>` tag and added the "javascript" tag to the icon object in `devicon.json`. If it needs any other changes let me know.

**Review by Snailedlt — APPROVED** [review]
Looks good to me.
Great work! 💯

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**